docs/testing-framework.md

```diff
@@ -0,0 +1,438 @@
+---
+title: Cadence Testing Framework
+---
+
+Cadence testing framework provides a convenient way to write tests for Cadence programs in Cadence.
+This functionality is provided by the Test standard-library.
+
+Note: Test standard-library can only be used off-chain. e.g: In CLI
+
+## Test Standard Library
+
+Testing standard library can be imported directly into the test script.
+```cadence
+import Test
+```
+
+## Assertion
+### assert
+```cadence
+fun assert(_ condition: Bool, _ message: String)
```

Comment:
This is an implementation different: to match the rest of the functions in the `Test` stdlib. But yeah, can unify both. We can also remove this one, since it's already there in other stdlib

Reply:
Updated in https://github.com/onflow/cadence/pull/1900/commits/f1e7a85f615371f218068328be93589db5db95c7 for the docs. The behaviour is also updated in the other PR.